PrestaShop/PrestaShop
# Migration of  "Design > Pages" page cms category listing

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Migration of  "Design > Pages" CMS pages. There are 2 listings in this page, this PR  **only migrates the top listing: categories** and the **breadcrumbs** related with it above. Page can be reached via **admin-dev/index.php/improve/design/cms-pages**
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10565
| How to test?  | See below

## How to test

There are 2 listings in this page. This PR migrates the top listing: categories. It also migrates thebreadcrumbs.
The **CMS category list** and breadcrumbs should work exactly the same as in legacy page (excluding position drag & drop action).
Please note that
- everything regarding the second listing
- positions drag and drop and search by position

will be migrated in another PR.
Access migrated page through `admin-dev/index.php/improve/design/cms-pages`




---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/10877)
